include/zephyr/usbc/usbc.h

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright 2022 The Chromium OS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief USBC Device APIs
+ *
+ * This file contains the USBC Device APIs.
+ */
+
+#ifndef ZEPHYR_INCLUDE_USBC_H_
+#define ZEPHYR_INCLUDE_USBC_H_
+
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/usbc/usbc_tcpc.h>
+#include <zephyr/drivers/usbc/usbc_vbus.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief USBC Device APIs
+ * @defgroup _usbc_device_api USBC Device API
+ * @{
+ */
+
+/**
+ * @brief This Request Data Object (RDO) value can be returned from the
+ *	  policy_cb_get_rdo if 5V@100mA with the following
+ *	  options are sufficient for the Sink to operate.
+ *
+ *	  The RDO is configured as follows:
+ *		Maximum operating current 100mA
+ *		Operating current 100mA
+ *		Unchunked Extended Messages Not Supported
+ *		No USB Suspend
+ *		Not USB Communications Capable
+ *		No capability mismatch
+ *		Don't giveback
+ *		Object position 1 (5V PDO)
+ */
```

Comment:
Feels like we need to namespace some of these enums and defines.  `policy_request_t` seems a bit too generic.

Reply:
I've prefixed the enums with "usbc_"